applications/rag/README.md

```diff
@@ -135,18 +135,20 @@ This step generates the vector embeddings for your input dataset. Currently, the
 
 4. Once logged in, choose the `CPU` preset. Go to File -> Open From URL & upload the notebook `rag-kaggle-ray-sql.ipynb` from `https://raw.githubusercontent.com/GoogleCloudPlatform/ai-on-gke/main/applications/rag/example_notebooks/rag-kaggle-ray-sql-latest.ipynb`. This path can also be found by going to the [notebook location](https://github.com/GoogleCloudPlatform/ai-on-gke/blob/main/applications/rag/example_notebooks/rag-kaggle-ray-sql-latest.ipynb) and selecting `Raw`.
 
-5. Replace the variables in the 3rd cell with the following to access the database:
-    * `INSTANCE_CONNECTION_NAME`: `<project_id>:<region>:pgvector-instance`
-    * `DB_USER`: `rag-user-notebook`
-    * `DB_PASS`: password from step 1
 
-6. Create a Kaggle account and navigate to https://www.kaggle.com/settings/account and generate an API token. See https://www.kaggle.com/docs/api#authentication how to create one from https://kaggle.com/settings. This token is used in the notebook to access the [Google Maps Restaurant Reviews dataset](https://www.kaggle.com/datasets/denizbilginn/google-maps-restaurant-reviews)
+5. Create a Kaggle account and navigate to https://www.kaggle.com/settings/account and generate an API token. See https://www.kaggle.com/docs/api#authentication how to create one from https://kaggle.com/settings. This token is used in the notebook to access the [Google Maps Restaurant Reviews dataset](https://www.kaggle.com/datasets/denizbilginn/google-maps-restaurant-reviews)
 
-8. Replace the kaggle username and api token in 2nd cell with your credentials (can be found in the `kaggle.json` file created by Step 6):
-    * `os.environ['KAGGLE_USERNAME']`
-    * `os.environ['KAGGLE_KEY']`
+6. Replace the variables in the 1st cell with custom values:
+    * Kaggle credentials (can be found in the `kaggle.json` file created by Step 5):
+        * `os.environ['KAGGLE_USERNAME']`
+        * `os.environ['KAGGLE_KEY']`
+    * DB credentials:
+        * `DB_REGION` = ${CLUSTER_REGION:?} # Note this needs to be the GCE region the instance is in (not zone).
```

Comment:
This implies it's being pasted into shell. Let's just enumerate the variables that need to be filled in:

* DB credentials:
  * `DB_REGION`
  * `PROJECT_ID`
  * `DB_PASS` (from step 1)

Additionally:
* Drop `DB_USER`, as most users won't change it and it has a good default.
* "GCE region (not zone) of the instance" is already spelled out in the notebook. Since they will be reading that comment anyways, I wouldn't duplicate the note in both places.